packages/clients/meldemichel/src/mapConfigurations.ts

```diff
@@ -123,8 +123,11 @@ const geoLocation: Partial<GeoLocationConfiguration> = {
   showTooltip: true,
 }
 
-const mapConfigurations = {
-  [MODE.COMPLETE]: (reportServiceId: string) => {
+const mapConfigurations: Record<
+  string,
+  (reportServiceId: string, afmUrl: string) => object
```

Comment:
Correct would be to return `MapConfig` here but that wouldn't immediately work because of the overlap of configuring a plugin during `addPlugins` or during `createMap` via this config.

An idea would be to have all the completely required parameters, meaning a plugin would not work if not present, to be present during initialization in `addPlugins` and make all configurations in `MapConfig` a `Partial`, so that parameters be be overwritten.

If you want, you can add that change here. Otherwise, feel free to create an issue and I will tackle that.

Reply:
https://github.com/Dataport/polar/issues/25